docs/hosting-a-server-with-northstar/guides/pg9182's-Docker-Installation-Guide.md

```diff
@@ -0,0 +1,84 @@
+| ------------------| ---------------------------------------------------------| 
+| Kernal            | Linux 5.3+, but 4.9+ should work                         |
+| CPU               | x86_64, at least 3 cores minimum                         |
+| RAM               | 2GB (Typically peaks at 1.6GB)                           |
+| Disk              | 5GB                                                      |
+| Network           | Recommended at least 7-20Mbps up                         |
+| Docker            | Have Docker and Docker-compose installed on the machine  |
+
+
+# <a name="Installation">Installation</a>
+
+## Prep Titanfall Server files
+You will need to copy over your titanfall installation folder and place it on the Linux machine. Most of these files are for SP and therefore we can delete from the installation folder and prune the file size down to ~5GB. 
+
+Easiest way to do this is copy the entire titanfall folder to your desktop and delete the [following files](https://github.com/pg9182/northstar-dedicated#reducing-the-size-). 
+
+## Copy Titanfall Folder over to the Linux Machine
+
+1. FTP into your linux machine using [Filezilla](https://filezilla-project.org/) or [WinSCP](https://winscp.net/eng/download.php) 
+2. Navigate to /mnt/titanfall (create directory if does not exist)
+3. Copy the newly pruned titanfall folder to the server. 
+
+![/mnt/titanfall/](https://i.postimg.cc/15HbbzFr/image.pnghttps://i.postimg.cc/15HbbzFr/image.png)
+
+## Copy mods to /mnt/mods
+If you have configured some mods,these can be placed at /mnt/mods like below
+![/mnt/mods](https://i.postimg.cc/tRD5jnrJ/image.png)
+
+## Create docker-compose file
+We are going to be using Docker-Compose to set up our container, this gives us much more flexibility and allows us to make changes to the start up args much cleaner. 
+ ```
+ mkdir Titanfall
+ nano docker-compose.yml
+ ```
+
+ Example docker-compose.yml
+
+ ```ruby
+ version: '3'
+ services:
+  northstar1: 
+    image: ghcr.io/pg9182/northstar-dedicated:1-tf2.0.11.0 
+    pull_policy: always 
+    environment:
+      - NS_PORT=37015
+      - NS_PORT_AUTH=8081
+      - 'NS_SERVER_NAME=[Region]Enter Server Name here'
+      - 'NS_SERVER_DESC=Enter your description here'
+      - |
+        NS_EXTRA_ARGUMENTS=
+        +setplaylist aitdm 
+        +mp_gamemode aitdm 
+        +map mp_angel_city
+        +ns_private_match_countdown_length 0
+        +ns_should_return_to_lobby 0
+        +net_compresspackets_minsize 64
+        +net_compresspackets 1
+        +spewlog_enable 0
+        +sv_maxrate 127000
+    volumes:
+      - /mnt/titanfall:/mnt/titanfall:ro
+      - /mnt/mods:/mnt/mods:ro
+    ports:
+      - '37015:37015/udp'
+      - '8081:8081/tcp'
+    restart: always
+ ```
+ A list of all the CONVARs are [here](https://r2northstar.gitbook.io/r2northstar-wiki/hosting-a-server-with-northstar/basic-listen-server#server-configuration)
```

Comment:
Should be a relative link, so

```suggestion
 A list of all the CONVARs are [here](../../basic-listen-server#server-configuration)
```

Reply:
Thats for the comments and suggestion, I'll put those through and PR again for review tonight.